airgun/entities/contentview_new.py

```diff
@@ -34,20 +40,57 @@ def search(self, value):
         view.wait_displayed()
         return view.search(value)
 
-    def publish(self, entity_name, values=None):
-        """Publishes new version of CV"""
+    def publish(self, entity_name, values=None, promote=False, lce=None):
+        """Publishes new version of CV, optionally allowing for instant promotion"""
         view = self.navigate_to(self, 'Publish', entity_name=entity_name)
         self.browser.plugin.ensure_page_safe(timeout='5s')
         view.wait_displayed()
         if values:
             view.fill(values)
-        view.next.click()
-        view.finish.click()
+        if promote:
+            view.promote.click()
+            view.lce_selector.fill({lce: True})
+        view.next_button.click()
+        view.finish_button.click()
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         self.browser.plugin.ensure_page_safe(timeout='5s')
         view.wait_displayed()
         return view.versions.table.read()
 
+    def add_content(self, entity_name, content_name):
+        """Add specified content to the given Content View"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        view.wait_displayed()
+        view.repositories.resources.add(content_name)
+        return view.repositories.resources.read()
+
+    def add_cv(self, ccv_name, cv_name, always_update=False, version=None):
+        """Adds selected CV to selected CCV, optionally with support for always_update and specified version"""
+        view = self.navigate_to(self, 'Edit', entity_name=ccv_name)
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        view.wait_displayed()
+        view.content_views.resources.add(cv_name)
+        view = AddContentViewModal(self.browser)
+        if always_update:
+            view.always_update.fill(True)
+        if version:
+            view.version_select.item_select(version)
+        view.submit_button.click()
+        view = self.navigate_to(self, 'Edit', entity_name=ccv_name)
+        return view.content_views.resources.read()
+
+    def read_cv(self, entity_name, version_name):
+        """Reads the table for a specified Content View's specified Version"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
```

Comment:
```suggestion
        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
        self.browser.plugin.ensure_page_safe(timeout='5s')
        view.wait_displayed()
```
I ran into intermittent local & PRT issue finding `view.versions` for search. 
Waiting until displayed resolved the issue from my draft Robo [PR 16283](https://github.com/SatelliteQE/robottelo/pull/16283),
 ( using draft airgun #1592, a rebase of this pr ) My draft's PRT still failed but not on `read_cv` anymore